source/Government.cpp

```diff
@@ -258,7 +263,21 @@ void Government::Load(const DataNode &node)
 		else if(key == "language")
 			language = child.Token(valueIndex);
 		else if(key == "raid")
-			raidFleet = GameData::Fleets().Get(child.Token(valueIndex));
+		{
+			const Fleet *raidingFleet = GameData::Fleets().Get(child.Token(valueIndex));
```

Comment:
If in the second definition of this government a raid is defined, it should overwrite the previous raid definitions, for backwards compatibility. Unless `add` is used ofc.

Reply:
it does, above?
if(raidnotcleared && !add && !remove) does that I think

Reply:
No, because that if above is guarded by if(remove). I'm talking in the scenario without any removes.

Reply:
oh yeah bruh what I had written was nonsense xD